tests/core/test_model.py

```diff
@@ -4186,7 +4186,7 @@ def test_default_catalog_sql(assert_exp_eq):
 
 
 def test_default_catalog_python():
-    HASH_WITH_CATALOG = "663490914"
+    HASH_WITH_CATALOG = "2071559843"
```

Comment:
i know you didn't write these but could we fix these hardcoded hash tests so they aren't hardcoded anymore

Reply:
It seems to me that these particular cases test exactly that, so we (1) can't know the hash/version before to do the assertion or (2) we do not want to propagate it as it could silently change. 

Though, I think we could modify the following:

- `tests/core/test_snapshot_evaluator.py`: Add `f"{snapshot.version}"` to the physical names

- `tests/core/test_state_sync.py`: The ordering here is affected by the values, so we could sort & replace them from least to greatest

Reply:
ok